UserExperience_Remove_CssJs_Page_View.js

```diff
@@ -27,21 +27,32 @@ jQuery(function() {
 				);
 
 				if (!exists) {
+					var single_id = single_path.replace(/[^\w-]/g, '_');
+
 					var li = jQuery(
-						'<li id="remove_cssjs_singles_' + single + '">' +
+						'<li id="remove_cssjs_singles_' + single_id + '">' +
 						'<table class="form-table">' +
 						'<tr>' +
 						'<th>CSS/JS path to remove:</th>' +
 						'<td>' +
-						'<span class="remove_cssjs_singles_path">' + single + '</span> ' +
-						'<input type="button" class="button w3tc_remove_cssjs_singles_delete" value="Delete" />' +
+						'<span class="remove_cssjs_singles_path">' + single_path + '</span>' +
+						'<input type="button" class="button remove_cssjs_singles_delete" value="Delete"/>' +
+						'</td>' +
+						'</tr>' +
+						'<tr>' +
+						'<th><label for="remove_cssjs_singles_' + single_id + '_action">Behavior:</label></th>' +
+						'<td>' +
+						'<label class="remove_cssjs_singles_behavior"><input class="remove_cssjs_singles_behavior_radio" type="radio" name="user-experience-remove-cssjs-singles[' + single_path + '][action]" value="exclude" checked>Exclude</label>' +
+						'<label class="remove_cssjs_singles_behavior"><input class="remove_cssjs_singles_behavior_radio" type="radio" name="user-experience-remove-cssjs-singles[' + single_path + '][action]" value="include">Include</label>' +
+						'<p class="description">Exclude will only remove this file from the specified URLs.</p>' +
+						'<p class="description">Include will NOT remove this file from the specified URLs but will remove it everywhere else.</p>' +
 						'</td>' +
 						'</tr>' +
 						'<tr>' +
-						'<th><label for="remove_cssjs_singles_' + single + '_includes">Remove on these pages:</label></th>' +
+						'<th><label class="remove_cssjs_singles_' + single_id + '_includes_label" for="remove_cssjs_singles_' + single_id + '_includes">Exclude on these pages:</label></th>' +
 						'<td>' +
-						'<textarea id="remove_cssjs_singles_' + single + '_includes" name="user-experience-remove-cssjs-singles[' + single + '][includes]" rows="5" cols="50"></textarea>' +
-						'<p class="description">Specify relative/absolute page URLs that the above CSS/JS should be removed from. Include one entry per line.</p>' +
+						'<textarea id="remove_cssjs_singles_' + single_id + '_includes" name="user-experience-remove-cssjs-singles[' + single_path + '][includes]" rows="5" cols="50" ></textarea>' +
+						'<p class="description remove_cssjs_singles_' + single_id + '_includes_description">Specify relative/absolute page URLs that the above CSS/JS file should be Excluded from. Include one entry per line.</p>' +
```

Comment:
All text should be run through translation functions.  You can do this in PHP and use https://developer.wordpress.org/reference/functions/wp_localize_script/ to pass an object to JavaScript.
Also, do not capitalize words like "Exclude" -- apply `<em>` or `<strong>` instead.  Change the wording "be excluded from" to "to be excluded".